tests/scripts/task_config_build_cpu.sh

```diff
@@ -34,7 +34,6 @@ echo set\(USE_NNPACK ON\) >> config.cmake
 echo set\(NNPACK_PATH /NNPACK/build/\) >> config.cmake
 echo set\(USE_ANTLR ON\) >> config.cmake
 echo set\(CMAKE_CXX_COMPILER g++\) >> config.cmake
-echo set\(CMAKE_CXX_FLAGS -Werror\) >> config.cmake
```

Comment:
Can we try pulling this back?..so others can see the issue in the CI log

Reply:
-Werror is present in all other configurations which don't build with Verilator. Pulled back it will cause lib_tsim.so compilation failure due to warnings in Verilator source code.

Reply:
Here is what worked for me. I removed verilated.cpp and verilated_dpi.cpp from cmake config src object list and included them in vta module.cc itself in 3rdparty/vta-hardware. Make compiles those sv functions right into module.cc object file where they are used. Somehow it doesn't flag warnings on compilation, but those needed functions got created. We will not see if it is the case till updated VTA is connected to this TVM. In this TVM those cpp files are not compiled and not required anywhere.
Disabling redefinition warning did not work as I was not able to find -W option for it. @stevenmburns